Libplanet/Net/NetMQTransport.cs

```diff
@@ -624,35 +624,46 @@ private void ReceiveMessage(object sender, NetMQSocketEventArgs e)
 
         private void DoBroadcast(object sender, NetMQQueueEventArgs<(Address?, Message)> e)
         {
-            (Address? except, Message msg) = e.Queue.Dequeue();
+            try
```

Comment:
I guess these changes is not related to this PR though...

Reply:
> *This must be merged after #1067 is merged first.*

I am going to rebase this on the current main!